client/src/components/UnsavedChangesIndicator/UnsavedChangesIndicator.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from 'i18n';
+import classnames from 'classnames';
+
+const UnsavedChangesIndicator = ({
+  level = 'none',
+  minutes = {
+    notice: 5,
+    warning: 10,
+  },
+}) => {
+  if (level === 'none') {
+    return null;
+  }
+  const isWarning = level === 'warning';
+  const mins = isWarning ? minutes.warning : minutes.notice;
+  let message;
+  let shortMessage;
+  if (mins <= 1) {
+    message = i18n.inject(
+      i18n._t('Admin.UNSAVED_CHANGES_SINGULAR_FULL', 'Unsaved changes: {mins} min'),
+      { mins }
+    );
+    shortMessage = i18n.inject(
+      i18n._t('Admin.UNSAVED_CHANGES_SINGULAR_SHORT', '{mins} min'),
+      { mins }
+    );
+  } else {
+    message = i18n.inject(
+      i18n._t('Admin.UNSAVED_CHANGES_PLURAL_FULL', 'Unsaved changes: {mins} mins'),
+      { mins }
+    );
+    shortMessage = i18n.inject(
+      i18n._t('Admin.UNSAVED_CHANGES_PLURA_SHORT', '{mins} mins'),
+      { mins }
+    );
+  }
+  const icon = isWarning ? 'font-icon-attention' : 'font-icon-info-circled';
+  const className = classnames('unsaved-changes-indicator', {
+    'unsaved-changes-indicator--notice': level === 'notice',
+    'unsaved-changes-indicator--warning': isWarning,
+  });
```

Comment:
```suggestion
  const className = classnames('unsaved-changes-indicator', `unsaved-changes-indicator--${level}');
```
`level` should always be one of "notice" or "warning" so we can just use string interpolation here. Probably don't even need to use `classnames()` tbh

Reply:
Updated